third_party/rust/chromium_crates_io/Cargo.lock

```diff
@@ -197,9 +216,9 @@ name = "bls-signatures"
 version = "0.12.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 dependencies = [
- "bls12_381",
+ "bls12_381 0.7.0",
```

Comment:
is there no way to resolve this so we only have one version of bls12_381? Same for group?

Reply:
Seems no, jubjub requires v0_8 and bls_signatures doesn't compile on v0_8.
For group halo_proofs requires v0_13 while bls_signature v0_12.

Reply:
we should create a follow-up here to see if we can bump bls-signatures to a version that gets rid of this and other crates with multiple versions here

Reply:
https://github.com/brave/brave-browser/issues/37680